ManatiCR/vitaminau
# VIT-78: Correct required text in form changes.

# Task [VIT-78](https://manati.atlassian.net/browse/VIT-78).
Changes in beneficiaries section in all forms.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following URL: 
  - http://vitaminau.docker/formularios/venta-de-productos
  - http://vitaminau.docker/formularios/proyecto-cultural-deportivo
  - http://vitaminau.docker/formularios/tecnologias-licenciables
- Check the messages of the required fields (El campo "Field_Name" es requerido):
  - Correo para solicitar información
  - Beneficiarios -> Persona individual -> Con especificaciones -> Especificaciones

# CA
- The text changes made must correspond to the input.
